python/generate/templates/model.mustache

```diff
@@ -1,243 +0,0 @@
-# coding: utf-8
```

Comment:
Where is the replacement for this template? Should there be a modelGeneric.mustache template? As per comments here (https://github.com/finbourne/lusid-sdk-python-preview/pull/105/files#) we have lost some functionality we had from this template.